src/Core/Collections/Leaderboard/components/SearchBar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ChakraProps, Input, InputGroup, InputRightElement } from '@chakra-ui/react';
+import * as Icons from '../../iconography';
+
+type Props = {
+  className?: string;
+  onChange?: (p: { value: string }) => void;
+  styles?: ChakraProps['sx'];
+  value?: string;
+};
+
+export const SearchBar: React.FC<Props> = ({ className, onChange, styles }) => {
+  const handleChange = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    onChange && onChange({ value: target.value });
```

Comment:
remove this onChange check

```suggestion
  onChange({ value: target.value });
```